src/shared/app-config.ts

```
@@ -0,0 +1,47 @@
+import packageInfo from "../../package.json" assert { type: "json" };
+
+class AppConfig {
+  private _domain: string = "707x.in";
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)